internal/nostr/nostr.go

```diff
@@ -116,6 +118,12 @@ func NewService(ctx context.Context) (*Service, error) {
 
 	subscriptions := make(map[string]*nostr.Subscription)
 
+	// TODO: Check limits
+	client := expo.NewPushClient(&expo.ClientConfig{
+		Host:  "https://api.expo.dev",
+		APIURL: "/v2",
+	})
+
```

Comment:
We have to check the api limitations, it also takes in an access token here

Reply:
What API limitations do you have in mind?

Reply:
I was not expecting this service to be free, but [expo docs](https://docs.expo.dev/push-notifications/faq/) do mention there is no cost. There's a limit of 600 notifications per second, but that should be quite good for us.